### PR TITLE
Add component Title / add package for Jest

### PR DESCRIPTION
**Pull request by Gachegov59**
- Create component Title + tests

### CODE REVIEW

File: classes/fullstack/project1/client/src/components/Title/Title.test.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { render } from '@testing-library/react';
```
Comment (Darkmift) [inline classes/fullstack/project1/client/src/components/Title/Title.test.tsx:1]: very good structure

File: classes/fullstack/project1/client/src/components/Title/Title.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { FC } from 'react';
+
+export type TitleProps = {
+  text: string;
+  size?: 1 | 2 | 3 | 4 | 5 | 6;
+  className?: string;
+};
+
+const Title: FC<TitleProps> = ({ text, size = 1, className = '' }) => {
```
Comment (Darkmift) [inline classes/fullstack/project1/client/src/components/Title/Title.tsx:9]: Very nice pattern!!

**Comment by Gachegov59** [comment]
I removed Jest from this PR and create new [issue](https://github.com/Darkmift/ELEVATION-ITWORKS/issues/57)

File: classes/fullstack/project1/client/src/App.tsx [hunk 3]
```diff
@@ -16,7 +17,7 @@ function App() {
           <img src={reactLogo} className="logo react" alt="React logo" />
         </a>
       </div>
-      <h1>Vite + React</h1>
+      <Title text="Vite + React" size={1} className='text-nowrap'/>
```
Comment (Darkmift) [inline classes/fullstack/project1/client/src/App.tsx:20]: please remove that as well

**Review by Darkmift — APPROVED** [review]

**Review by Darkmift — APPROVED** [review]

**Pull request merged**